src/Hyperlink.js

```diff
@@ -52,10 +59,10 @@ class Hyperlink extends Component {
     return typeof component.props.children !== 'string'
   }
 
-  linkify(component){
+  linkify = (component) => {
```

Comment:
The babel build script failed due to this change; transforming this new class property syntax requires the babel plugin [proposal-class-properties](https://babeljs.io/docs/en/babel-plugin-proposal-class-properties), which requires Babel 7